sdk/src/sharedTest/java/co/omise/android/ui/CreditCardActivityTest.kt

```diff
@@ -102,7 +102,7 @@ class CreditCardActivityTest {
     fun form_validForm() {
         onView(withId(R.id.edit_card_number)).perform(typeText("4242424242424242"))
         onView(withId(R.id.edit_card_name)).perform(typeText("John Doe"))
-        onView(withId(R.id.edit_expiry_date)).perform(typeText("1234"))
+        onView(withId(R.id.edit_expiry_date)).perform(typeNumberText("1234"))
```

Comment:
@AnasNaouchi maybe one of the conditions we're not testing is `s.length > MAX_CHARS`